### PR TITLE
Fix README, launch

### LINKED ISSUE #48: Fix typo in readme
suod

### PR DESCRIPTION
**Pull request by kevin1055**
Fixes #48

### CODE REVIEW

File: launch [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #!/bin/bash -x
-kill $(cat lendbot.pid)
 venv/bin/pip3 install -r requirements.txt
 nohup venv/bin/python3 launcher.py & disown
 echo $! > lendbot.pid
```
Comment (blerud) [inline launch]: can remove this line too

Comment (kevin1055) [reply launch]: ah yes

File: launch [hunk 2]
```diff
@@ -1,5 +1,3 @@
 #!/bin/bash -x
-kill $(cat lendbot.pid)
 venv/bin/pip3 install -r requirements.txt
 nohup venv/bin/python3 launcher.py & disown
```
Comment (blerud) [inline launch]: dont need the disown either right

Comment (kevin1055) [reply launch]: idk best leave it in man who knows

**Pull request merged**